packages/sanity/src/core/form/members/object/fields/ObjectField.tsx

```diff
@@ -34,6 +37,11 @@ export const ObjectField = function ObjectField(props: {
   renderItem: RenderArrayOfObjectsItemCallback
   renderPreview: RenderPreviewCallback
 }) {
+  const {member} = props
+  const {__internal, focusPath} = useFormBuilder()
+  const {documentId, field} = __internal
+  const focused = pathToString(focusPath) === pathToString(member.field.path)
```

Comment:
Replace this with `member.field.focused`

Reply:
Ah, nice – much better!